build.sbt

```diff
@@ -25,6 +25,9 @@ organization := "com.linkedin.drelephant"
 
 javacOptions in Compile ++= Seq("-source", "1.6", "-target", "1.6")
 
+// Prevent failure on some environment
```

Comment:
More specifically, 7 tests fails with the error : 
`[error] Test com.linkedin.drelephant.util.MemoryFormatUtilsTest.testBytesToString failed: expected:<1[,]000 GB> but was:<1[ ]000 GB>`
on a laptop with LANG=fr_FR.UTF-8.

So indeed, changing the LANG env variable should solve the issue, but this require the user to change his settings, which he may not necessarily want. Is my solution not cleaner ? There may be something I am missing here, not really my comfort zone.